timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -228,27 +228,30 @@ types:
       LeaderTimes:
         fields:
           leaderTimes: map<Namespace, LeaderTime>
-      TimestampLeaseRequest:
-         fields:
-           requestId:
-             type: uuid
-             safety: safe
-           numFreshTimestamps:
-             type: integer
-             safety: safe
+      RequestId:
+        alias: uuid
+        safety: safe
       TimestampLeaseRequests:
-         alias: map<TimestampLeaseName, TimestampLeaseRequest>
+          fields:
+            requestsId: RequestId
+            numFreshTimestamps: map<TimestampLeaseName, integer>
+      TimestampLeasesRequest:
+          alias: list<TimestampLeaseRequests>
```

Comment:
This is many transactions. It's fine to use a list because the UUID are unique anyway. This will be the unit for any later single-client-batching.

Bear in mind if we get there that the request sizes for the batches may get big.

Reply:
Yep, I think for now we can constraint the autobatcher max batch size and call it a day.